packages/web-app-files/tests/unit/components/Collaborators/Collaborator.spec.js

```diff
@@ -183,11 +193,154 @@ describe('Collaborator component', () => {
         )
       })
     })
+    describe('reshare information', () => {
+      it.each([undefined, 0, null, false, '0', '', []])(
+        "does not show information if it's not a reshared share",
+        resharers => {
+          const wrapper = createWrapper({
+            shareType: 0,
+            collaborator: {
+              name: 'alice'
+            },
+            currentUserId: 'carol',
+            resharers: resharers
```

Comment:
Could be re-used more often below (also by accessing specific resharers through `resharers[x]` e.g. :)

Reply:
not in this particular test, because here `resharers` are invalid data coming from `it.each` but generally you are right. I've created a global reshares const to be reused